app/client/components/payment/directDebitLogo.tsx

```diff
@@ -1,84 +1,26 @@
+import { css } from "@emotion/core";
 import React from "react";
 
 export interface DirectDebitLogoProps {
   fill?: string;
   justLogo?: true;
+  marginRight?: true;
 }
 
-export const DirectDebitLogo = (props: DirectDebitLogoProps) =>
```

Comment:
This looks quite a drastic reduction in LoC to represent the DD logo, which is great but I think we need some screenshots of all the places the DD logo is used (e.g. in the DD guarantee on the payment update flow) to make sure it still looks fine?

Reply:
Updated description with more screenshots